scripts/intfutil

```diff
@@ -41,20 +46,29 @@ VLAN_SUB_INTERFACE_TYPE = "802.1q-encapsulation"
 
 SUB_PORT = "subport"
 
-def db_connect_configdb():
+
+def db_config_unload():
+    swsssdk.SonicDBConfig._sonic_db_global_config_init = False
```

Comment:
_variable_names are intended to be used as private variables. Suggest writing setter decorator.